src/main/java/kvp/hhlin/minifullfillment/domain/Center.java

```diff
@@ -0,0 +1,40 @@
+package kvp.hhlin.minifullfillment.domain;
+
+public class Center {
+
+    private String id;
+
+    private String name;
+
+    private String status;
```

Comment:
status는 Enum값을 그대로 가지고 있어도 괜찮을 것 같아요!!

```suggestion
    private CenterStatus status;
```